mw-webapp/src/logic/allUsersPage/AllUsersPage.module.scss

```diff
@@ -15,11 +15,36 @@
   justify-content: space-between;
 }
 
-.inputFilter {
+.inputWrapper {
+  position: relative;
+  width: 100%;
   max-width: $maxWidthBlock;
-  padding: $paddingSmall;
+}
+
+.inputIcon {
+  position: absolute;
+  left: 5px;
+  top: 50%;
+  transform: translateY(-50%);
+  fill: var(--inactiveTextColor);
+  stroke: none;
+}
+
+.inputFilter {
+  padding: $paddingSmall $paddingSmall $paddingSmall 25px;
```

Comment:
1.
Please don't use magic numbers.
2. Is it possible don't use padding-right: 25px? 
3. You can use short syntax.

Reply:
fixed